src/redux/actions/actionCreators.js

```diff
@@ -14,39 +14,27 @@ import {
   TOGGLE_FOLLOWING_PROGRESS,
   SET_USER_PROFILE,
   SET_USER_DATA,
+  SET_STATUS,
 } from './actionTypes';
 
 import { usersAPI, authAPI } from '../../api/api';
 
-const sendMessageCreator = () => ({ type: SEND_MESSAGE });
-
-const updateNewMessageBodyCreator = (body) => ({
-  type: UPDATE_NEW_MESSAGE_BODY,
-  body,
-});
 
 const addPostActionCreator = () => ({ type: ADD_POST });
-
-const updateNewPostTextActionCreator = (text) => ({
-  type: UPDATE_NEW_POST_TEXT,
-  newText: text,
-});
-
+const setUsers = (users) => ({ type: SET_USERS, users });
+const sendMessageCreator = () => ({ type: SEND_MESSAGE });
+const setStatus = (status) => ({ type: SET_STATUS, status });
 const followSuccess = (userID) => ({ type: FOLLOW, userID });
 const unfollowSuccess = (userID) => ({ type: UNFOLLOW, userID });
-const setUsers = (users) => ({ type: SET_USERS, users });
+const setUserProfile = (profile) => ({ type: SET_USER_PROFILE, profile });
 const setCurrentPage = (currentPage) => ({ type: SET_CURRENT_PAGE, currentPage });
-const setTotalUsersCount = (totalUsersCount) => ({ type: SET_TOTAL_USERS_COUNT, totalUsersCount });
 const toggleIsFetching = (isFetching) => ({ type: TOGGLE_IS_FETCHING, isFetching });
-const toggleFollowingProgress = (isFetching, userID) => ({
-  type: TOGGLE_FOLLOWING_PROGRESS, isFetching, userID,
-});
-const setUserProfile = (profile) => ({ type: SET_USER_PROFILE, profile });
-
+const updateNewMessageBodyCreator = (body) => ({ type: UPDATE_NEW_MESSAGE_BODY, body });
```

Comment:
It's a good practice to make a uniformed action object - typically with "type" and "payload" fields in it.
This way you'll have a generic structure of each of your actions.
Therefore, consider reworking every object you have to support the following structure:
`{ type: '...', payload: {...}, }`